config/config.go

```diff
@@ -53,6 +53,14 @@ type Config struct {
 	COAKey crypto.PrivateKey
 	// COAKeys is a slice of all the keys that will be used in key-rotation mechanism.
 	COAKeys []crypto.PrivateKey
+	// COACloudKMSKeys is a slice of all the keys and their versions that will be used in Cloud KMS key-rotation mechanism.
+	COACloudKMSKeys []string
+	// COACloudKMSProjectID is the project ID containing the KMS keys
+	COACloudKMSProjectID string
+	// COACloudKMSLocationID is the location ID where the key ring is grouped into
+	COACloudKMSLocationID string
+	// COACloudKMSKeyRingID is the key ring ID where the KMS keys exist
+	COACloudKMSKeyRingID string
```

Comment:
should be of different type, check the other comment

Reply:
Changed in https://github.com/onflow/flow-evm-gateway/pull/282/commits/9a3e78e3e08b3eb030231b4f4df8fc373bbbbfb5#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R58